Herebyfile.mjs

```diff
@@ -294,21 +294,28 @@ async function runTestTools() {
     await $test({ cwd: path.join(__dirname, "_tools") })`${gotestsum()} ./...`;
 }
 
+async function runTestAPI() {
+    await $`npm run -w @typescript/api test`;
+}
+
 export const testTools = task({
     name: "test:tools",
     run: runTestTools,
 });
 
-export const testAll = task({
-    name: "test:all",

```

Comment:
On second thought, maybe we shouldn't drop this task entirely (just not use it in CI). But that might be challenging to get right due to deps / getting them to run in order?

Reply:
Related, do you want to run the API benchmarks as part of `test:benchmarks`? They’re currently not running as part of a hereby script.

Reply:
I'm not sure; for the Go tests, they're effectively just "more tests" we can opt to run a specific number of time.

I think the API stuff is sort of separate, but I can see it being part of the test suite for `test:api:benchmarks` or something if we are able to run them once just to verify that they don't crash. (That's the main reason to check them `1x`, and indeed we caught a bug in the benchmarks via that check just today.)